server/src/main/java/org/elasticsearch/search/MultiValueMode.java

```diff
@@ -20,12 +20,7 @@
 
 package org.elasticsearch.search;
 
-import org.apache.lucene.index.BinaryDocValues;
-import org.apache.lucene.index.DocValues;
-import org.apache.lucene.index.NumericDocValues;
-import org.apache.lucene.index.SortedDocValues;
-import org.apache.lucene.index.SortedNumericDocValues;
-import org.apache.lucene.index.SortedSetDocValues;
+import org.apache.lucene.index.*;
```

Comment:
we disallow wildcard import in our build

Reply:
Sorry about that.
I think IntelliJ did it.
I will fix.